src/contracts/libraries/GPv2Encoding.sol

```diff
@@ -20,6 +20,13 @@ library GPv2Encoding {
         bool partiallyFillable;
     }
 
+    /// @dev A struct representing arbitrary contract interactions.
+    /// Submitted to GPv2Settlement.settle for code execution.
```

Comment:
We've been writing the doc comments in Markdown. In that vain, maybe this could be a link?
```suggestion
    /// Submitted to [`GPv2Settlement.settle`] for code execution.
```